cmd/antictl/main.go

```diff
@@ -125,7 +126,99 @@ func main() {
 
 					},
 				},
-				// TODO (mierdin): Add command to make a session persistent
+				{
+					Name:  "persist",
+					Usage: "Make a LiveSession persistent",
+					Action: func(c *cli.Context) {
+						conn, err := grpc.Dial(fmt.Sprintf("%s:%s", host, port), grpc.WithInsecure())
+						if err != nil {
+							fmt.Println(err)
+							os.Exit(1)
+						}
+						defer conn.Close()
+						client := pb.NewLiveSessionsServiceClient(conn)
+
+						if len(c.Args()) < 2 {
+							fmt.Println("Missing args to command : antictl livesession persist <true/false> <session id>")
+							os.Exit(1)
+						}
+
+						persistent, err := strconv.ParseBool(c.Args()[0])
```

Comment:
Just a UX concern - the current order of the arguments seems a bit clunky:

```
antictl livesession persist true abcdef
```

Given the approach of the current function, I think the best way of doing it would be to reverse the two args so that you provide the session ID first.

```
antictl livesession persist abcdef true
```

Reply:
Agreed, changed the order of args and submitted